lib/App/cpanminus/script.pm

```diff
@@ -91,6 +91,7 @@ sub new {
         installed_dists => 0,
         install_types => ['requires'],
         with_develop => 0,
+        with_configure => undef,
```

Comment:
Other approaches to this that come to mind are 

1) split up the state of with-configure and without-configure to be 2 separate booleans

2) remove the --without-configure option (mostly there to continue the pattern from --with-develop/--without-develop)

3) make cpanm's default behaviour consistent when reading dependencies from META files or a cpanfile  

Each approach has it's relative merits, however I felt the current approach followed the pattern from --with-develop/--without-develop, maintaining the current default behaviour in the simplest way. I'm happy to rework things however if you have a different preference or suggestion?